setup.py

```diff
@@ -31,6 +32,6 @@
     py_modules=['memory_profiler'],
     scripts=['mprof'],
     classifiers=[_f for _f in CLASSIFIERS.split('\n') if _f],
-    license='BSD'
-
+    license='BSD',
+    install_requires=['asyncio>=3.4.3'] if sys.version_info < (3, 4) else [],
```

Comment:
What about using PEP 426 markers instead? https://blog.ionelmc.ro/presentations/packaging/#slide:25

Reply:
@apollovy Hi. Are you going to finish that?